src/appleseed/foundation/meta/tests/test_imagefilewriter.cpp

```diff
@@ -0,0 +1,245 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2017-2018 Esteban Tovagliari, The appleseedhq Organization
```

Comment:
Change to `// Copyright (c) 2020 ChinYing-Li, The appleseedhq Organization`

Reply:
sorry for overlooking this, I will change that.